CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-### 0.3.1 (Next)
+### (Next)
 
-* [#43](https://github.com/ashkan18/graphlient/pull/38): Allow to load and dump schema to json - [@povilasjurcys](https://github.com/povilasjurcys).
-* Your contribution here.
+* [#45](https://github.com/ashkan18/graphlient/pull/45): Drop Support for Ruby 2.2 and Lock RuboCop - [@jonallured](https://github.com/jonallured)
```

Comment:
Danger is complaining about the missing period here at the end.

Reply:
Thanks, I had no idea what it was complaining about, haha - fixed with dbdb679.